### PR TITLE
Feat: modfied oligo_extract.py to extract oligo entities

### PR DESCRIPTION
**Pull request by malayjoshi13**

### CODE REVIEW

File: oligo_extract.py [hunk 1]
```diff
@@ -65,21 +65,231 @@ def get_paper_sentences_with_TE(wbpids, settings):
...
+            # Here "|" means "or"
+            if (re.search("\S+@\S+\.\S+|https:\/\/|www.", clean_row)):
                 continue
 
             paperid_sentence_list.append((id, clean_row))
     return paperid_sentence_list 
 
+#................................................................
+
+def fn1(word, final_output, part, output = None, last_output = None):
```
Comment (abecerra) [inline oligo_extract.py]: Please name your functions describing what they do

File: oligo_extract.py [hunk 2]
```diff
@@ -65,21 +65,231 @@ def get_paper_sentences_with_TE(wbpids, settings):
...
+                else:
+                    output = word
+                    if output:
+                        last_output = output                    
+
+    return(output, last_output)
+
+#................................................................
+
+def fn2(word, final_output, part, output = None, last_output = None):
```
Comment (abecerra) [inline oligo_extract.py]: Please name your functions describing what they do

File: variants.csv [hunk 3]
```diff
@@ -0,0 +1,10 @@
+WBPaper ID,Oligo,Sentence
```
Comment (abecerra) [inline variants.csv]: Please remove example outputs like this csv file

Comment (abecerra) [reply variants.csv]: You can also remove folders: data, models and most of utils

Comment (malayjoshi13) [reply variants.csv]: These files are been currently used in text extraction and settings file, so cannot remove them.

Comment (abecerra) [reply variants.csv]: They can be removed if you remove the settings.py file and pass the token only (perhaps loaded from an environmental variable?

**Review by abecerra — CHANGES_REQUESTED** [review]
Could you please remove settings.py as well, and pass the textpresso token where needed?
Token could be read from an environment variable in a small function ... or from the utils/all_config.cfg ...

This could be done in a read_token function. Probably the all_config.cfg is the most similar to the other pipeline

**Pull request merged**